pages/snowdepth/add.vue

```diff
@@ -200,6 +206,25 @@ export default {
 
       L.control.layers(null, markerOverlays).addTo(newMap)
     }
+
```

Comment:
Please add a comment about this sorcery ... What are you extending and why?

Reply:
Added a comment above to explain what we're extending and why.